README.md

```diff
@@ -1,31 +1,403 @@
+  if (isProduction) {
+    config.mode = "production";
+  } else {
+    config.mode = "development";
+  }
+  return config;
+};
+```
+12. Change the `index.ts` file extention to `index.tsx` and edit the `index.tsx` file to match the following example:
+```js
+
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+
+const App = () => {
+  return (
+    <div>I am an app</div>
+  )
+}
+
+const domNode = document.getElementById('root');
+if(domNode) {
+  const root = createRoot(domNode);
+  root.render(<App />)
+}
+
+```
+
+If everything is set up, you can run `npm run serve` and view the example text that you entered in the `<div>` of `index.tsx`. 
+
+### Scalprum setup 
+
+To use Scalprum, we must create a **host application**.
+
+A *host application* is the main provider that manages module loading, routing, data sharing, and related tasks. It functions as the manager of the micro-frontend. 
+The host application always loads first. 
+It can be any React application as long as it has some mandatory webpack configuration.
+
+1. To create a host application - a top-level application to manage the micro frontend, in the `scalprum-demo/src` directory, create a `ScalprumRoot.tsx` file, and edit to match the following configuration: 
+```jsx
+
+import React from 'react';
+import { AppsConfig } from '@scalprum/core'
+import { ScalprumProvider, ScalprumComponent } from '@scalprum/react-core'
+
+const config: AppsConfig = {
+  remoteModule: {
+    name: 'remoteModule',
+    manifestLocation: 'http://localhost:8003/plugin-manifest.json'
+  }
+}
+
+const ScalprumRoot = () => {
+  return (
+    <div>
+      <ScalprumProvider config={config}>
+        <ScalprumComponent scope="remoteModule" module="RemoteModuleComponent" />
+      </ScalprumProvider>
+    </div>
+  )
+}
+
+export default ScalprumRoot;
+```
+
+Let's take a look at what we set up in this example. 
+
+
+
+| Component | Definition | 
+| -------- | -------- | 
+| **`AppsConfig module`**     | The **`AppsConfig`** is a set of dynamic modules from '@scalprum/core'. This is framework-agnostic and will work as long as it is in Javascript.      | 
+|**`@scalprum/core`** package| The **`@scalprum/core`** package is responsible for managing  federated modules. It provides an abstraction on low-level API to make it more developer friendly alongside additional features. Behind the scenes, it uses our exposed webpack modules, **`@openshift/dynamic-plugin-sdk`**, to manage plugins.|
+|**`ScalprumProvider`** module|The **`ScalprumProvider`** is the main root component. If you want to load modules, you must ensure they are enclosed in the `<ScalprumProvider />` tags. `ScalprumProvider` has a mandatory `config` field. This informs `ScalprumProvider` of which remote modules are available to render. For the purpose of the demo, `const config: AppsConfig = {}`, we set up `remoteModule` to use in the next steps. The config requires a `name`, and a `manifestLocation`.|
```

Comment:
In react we the elements are not generally described as `tags` but as components.

Reply:
Thank you.